src/main/java/visualizations/PieChart.java

```diff
@@ -0,0 +1,23 @@
+package visualizations;
+
+import tech.tablesaw.api.Table;
+import tech.tablesaw.plotly.Plot;
+import tech.tablesaw.plotly.api.PiePlot;
+
+/**
+ * Class that plots basic Pie charts.
+ */
+public class PieChart {
```

Comment:
```PiePlot``` I think is better

Reply:
it's taken from tablesaw.
I agree it's better but...